Presentation.Web/Controllers/API/V2/Common/Mapping/IOrganizationResponseMapper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Core.ApplicationServices.Model.Organizations;
+using Core.DomainModel.Organization;
+using Presentation.Web.Models.API.V2.Internal.Response.Organizations;
+using Presentation.Web.Models.API.V2.Response.Organization;
+
+namespace Presentation.Web.Controllers.API.V2.External.Generic;
+
+public interface IOrganizationResponseMapper
+{
+    OrganizationResponseDTO ToOrganizationDTO(Organization organization);
```

Comment:
The file paths differ (maybe a bug), and it should be placed in the Internal/Organizations/Mapping (same goes for other mappers)
![image](https://github.com/user-attachments/assets/79b0b317-e1ea-4d91-9a85-4b8d30a29150)

Reply:
I see now it's used in both external and internal. Maybe then Common/Mapping/Organizations/?

Reply:
Creating /Orgs caused trouble when refactoring automatically, decided it's not worth the work compared to leaving them in /mapping